src/makeCalculator.js

```diff
@@ -4,7 +4,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculate = {
+    result: 0,
+
+    add(result, number) {
+      result += number;
+    },
+
+    subtract(result, number) {
+      result -= number;
+    },
+
+    multiply(result, number) {
+      result *= number;
+    },
+
+    divide(result, number) {
+      result /= number;
```

Comment:
The 'divide' method should alter the 'result' property of the 'calculate' object. Use 'this.result' in place of the local 'result' parameter.